src/nodes/linkBlock.ts

```diff
@@ -0,0 +1,80 @@
+import { MyNodeSpec, NodeGroups } from './types';
+import { MdFormatSerialize, TexFormatSerialize } from '../serialize/types';
+import { StaticPhrasingContent, LinkBlock } from '../spec';
+
+export interface Attrs {
+  title: string;
+  description: string;
+  url: string;
+}
+
+const link_block: MyNodeSpec<Attrs, LinkBlock> = {
+  attrs: {
+    url: { default: '' },
+    title: { default: '' },
+    description: { default: '' },
+  },
+  group: NodeGroups.top,
+  content: `${NodeGroups.text}*`,
+  parseDOM: [
+    {
+      tag: 'div[title][data-url]',
```

Comment:
Let's put this on with a class name, and up the `priority` in the parsing. This has to match the `toDOM` method below.